src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -1000,6 +1002,18 @@ private CaseDTO createNewCachedCaseResponse(CachedCase newCase) throws CTPExcept
     return response;
   }
 
+  private void checkCaseTypeForNewAddress(
+      CaseType caseType, uk.gov.ons.ctp.integration.contactcentresvc.representation.Region region)
+      throws CTPException {
+    if ((caseType.name().equals("CE")) && (region.name().equals("N"))) {
+      String message =
+          "All queries relating to Communal Establishments in Northern Ireland "
+              + "should be escalated to NISRA HQ";
+      log.with(caseType.name()).with(region.name()).warn(message);
+      throw new CTPException(Fault.BAD_REQUEST, message);
+    }
+  }
+
   private void checkCaseIsNotTypeCE(CaseContainerDTO caseDetails) throws CTPException {
     if (caseDetails.getCaseType().equals("CE")) {
```

Comment:
While you are it please replace this string literal as well. Suggest use of

CaseType.valueOf(caseDetails.getCaseType()) == CE

ie use the OOTB enum functionality to turn the arriving string into the corresponding enum then compare that with an enum value

Reply:
Here I get the message "CE cannot be resolved to a variable"

Reply:
sorted now - thanks Phil